AEPServices/Sources/ui/fullscreen/FullscreenMessage.swift

```diff
@@ -120,12 +124,16 @@
 
             DispatchQueue.main.async {
 
-                // add observer to handle device rotation
                 if !self.observerSet {
-                    NotificationCenter.default.addObserver(self,
-                                                           selector: #selector(self.handleDeviceRotation(notification:)),
-                                                           name: UIDevice.orientationDidChangeNotification,
-                                                           object: nil)
```

Comment:
No need to for listen for notification : UIDevice.orientationDidChangeNotification
Because windowFrameObserver will also gets triggered when the device orientation changes